coffea/nanoevents/transforms.py

```diff
@@ -335,3 +335,7 @@ def nestedindex(stack):
 def item(stack):
     field = stack.pop()
     stack.append(stack.pop()[field])
+
+
+def to_numpy(stack):
```

Comment:
need this to put offsets that are directly popped of an existing `ListOffsetArray` into `local2global`

Reply:
As mentioned before, I think its fine to wrap the input of `local2global` with `numpy.array()` to reduce the need for this extra command